master/buildbot/schedulers/basic.py

```diff
@@ -62,6 +63,7 @@ def __init__(self, name, shouldntBeSet=NotSet, treeStableTimer=None,
         base.BaseScheduler.__init__(self, name, builderNames, properties, **kwargs)
 
         self.treeStableTimer = treeStableTimer
+        self.coolDownTimer = coolDownTimer
```

Comment:
while we follow the previous "tradition" of keeping all those value public, I think it'd be better if we make `coolDownTimer` private (= `self._coolDownTimer`)